clients/src/main/java/org/apache/kafka/clients/producer/MockProducer.java

```diff
@@ -377,6 +377,11 @@ public synchronized void clear() {
         this.producerFenced = false;
     }
 
+    public synchronized void setProcessingGuaranteeExactlyOnce() {
```

Comment:
I don't think we need to method -- we should rather call `initTranaction()` and `beginTransaction()` in the test driver

Reply:
Fixed